website/docusaurus.config.js

```diff
@@ -0,0 +1,129 @@
+/**
```

Comment:
Please use 2 spaces (at the most, 4) for indentation. Recommend you install the Prettier plugin for your editor.